_toc.yml

```diff
@@ -85,7 +85,6 @@ entries:
           - file: docs/platform/howto/use-billing-groups
           - file: docs/platform/howto/change-billing-contact
           - file: docs/platform/howto/payment-issues-plan-upgrades
-          - file: docs/platform/howto/custom-plans
```

Comment:
Why is this entry removed from `_toc`?

Reply:
I do not remember that I removed this entry. Only thing I did for this doc is adding a new entry. Let me add back.